types/public/Interfaces.d.ts

```diff
@@ -9,12 +9,18 @@
  * @returns Output bindings (optional). If you are returning a result from a Promise (or an async function), this
  * result will be passed to JSON.stringify unless it is a string, Buffer, ArrayBufferView, or number.
  */
-export declare type AzureFunction = ((context: Context, ...args: any[]) => Promise<any> | void);
+export declare type AzureFunction<TReturn = any, TArgs extends any[] = any[]> = ((context: Context, ...args: TArgs) => Promise<TReturn> | void);
```

Comment:
```suggestion
export declare type AzureFunction<TReturn = unknown, TArgs extends unknown[] = unknown[]> = ((context: Context, ...args: TArgs) => Promise<TReturn> | void);
```

These can be `unknown` instead of `any` :)